receiver/dockerstatsreceiver/docker_test.go

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !windows
 // +build !windows
+
```

Comment:
Please remove these as unrelated changes. Feel free to submit another PR to add the new style build comments